src/main/target/CRAZYBEEF3FR/target.h

```diff
@@ -134,7 +134,7 @@
 #define USE_RX_FRSKY_SPI_X
 #define USE_RX_SFHSS_SPI
 #define USE_RX_REDPINE_SPI
-#define DEFAULT_RX_FEATURE      FEATURE_RX_SPI	
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SPI  
```

Comment:
Nice try for converting extra space. 😝

Reply:
I blame my pre-commit detabbing script...